test/Twig/Tests/EnvironmentTest.php

```diff
@@ -180,13 +180,12 @@ public function testExtensionsAreNotInitializedWhenRenderingACompiledTemplate()
 
     public function testAutoReloadCacheMiss()
     {
-        $template_name = __FUNCTION__;
-        $template_content = __FUNCTION__;
+        $templateName = __FUNCTION__;
+        $templateContent = __FUNCTION__;
```

Comment:
ah sorry, maybe those lines come from the commit of @fabpot , which is in this PR?